content/git/concepts/diff/diff.md

```diff
@@ -0,0 +1,125 @@
+---
+Title: 'Diff'
+Description: 'A display of changes in different elements of a working environment: files, directories, commits, branches, etc.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'GitHub'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+In Git, __`diff`__ displays changes in different elements of your working environment: files, directories, commits and even branches. Simply put, diff shows what changed for one or several elements in a repository.
+
+## Syntax
+
+The syntax for comparing data changes in a repository using `diff` is:
+
+```bash
+$ git diff
+```
+## Examples
+
+The are various ways in which `git diff` can be used.
+
+### 1. git diff
+
+Used as it is, the command shows the difference between the working directory and the staging area.
+
+Suppose the initial version of the file _scene.txt_ below has already been [committed](https://www.codecademy.com/resources/docs/git/commit).
+
+```
+COBBYTO
+In a dream, brain activity will be roughly thirty times higher than usual. The effect is increased by entering a dream within that dream.
+
+ARIANEA
+How long should each level take?
+```
+
+Let's imagine we modify _scene.txt_ as followed and save the change in our working directory.
```

Comment:
```suggestion
Let's imagine we modify `scene.txt` as followed and save the change in our working directory.
```

Reply:
then you need see the if it is showing in staging area or not and then add this file using git add filename and and commit it after that if you want to push this file to your remote push it